site/en/docs/extensions/reference/storage/index.md

```diff
@@ -15,14 +15,22 @@ The Storage API provides an extension-specific way to persist user data and stat
 
 {% Details %}
 {% DetailsSummary %}
-💡 Can extensions use the WebStorage API?
+💡 Can extensions use the Storage API?
 {% endDetailsSummary %}
 
-Even though extensions can access [`Window.localStorage`][mdn-localstorage] in some contexts (popup and other HTML pages), it is not recommended for the following reasons:
+Even though extensions can use the [`Storage`][mdn-storage] interface (accessible from `window.localStorage`) in some contexts (popup and other HTML pages), it is not recommended for the following reasons:
 
-- The extension's service worker cannot access `Window.localStorage`.
-- Content scripts share the `Window.localStorage` of the host page.
-- Data saved to `Window.localStorage` is lost when the user clears their browsing history.
+- Extension's service worker cannot access `Storage`.
+- Content scripts share storage with the host page.
+- Data saved using the `Storage` interface is lost when the user clears their browsing history.
+
+Even though extension service workers can use `chrome.storage`, if need to move data, from the `Storage` interface to `chrome.storage` you'll need to use an [offscreen document][offscreen-document]. 
```

Comment:
Starting this with "even though" made me expect something like "you shouldn't use this". Could we re-word maybe to just be like "If you previously used the Storage interface and need to migrate to extension storage..."?

Reply:
Also in a suggestions.